src/main/webapp/app/shared/nav/StickyMiniNavBar.tsx

```diff
@@ -161,11 +161,11 @@ export default function StickyMiniNavBar({
   )[0]?.id;
 
   return (
-    <Container
+    <div
       className={classNames(
-        'container',
         styles.container,
-        isSticky ? styles.containerSticky : ''
+        isSticky ? styles.containerSticky : '',
+        'container'
```

Comment:
Whats the difference between the class and the component?

Reply:
Good question. Initially I removed the container, then added back the class. Let me try to revert to the component.